backend/model_manager.py

```
@@ -603,13 +603,15 @@ def download_file(url, filename, model_id, total_bytes=0):
             }
 
     except Exception as e:
-        _logger.error("Download failed: %s", e)
+        _logger.error("Download failed: %s", e, exc_info=True)
```

Ruff (0.15.21)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)